samples/name_domains_values/Coded_Domain_METOC_Space_Point_Entities.csv

```diff
@@ -1 +0,0 @@
-Name,Value
```

Comment:
This was a Pro problem for Dan, nothing JMSML-related (& removing empty domains sounds like a good idea)